logstash-core/src/main/java/org/logstash/ackedqueue/Queue.java

```diff
@@ -849,7 +855,7 @@ public long getUnackedCount() {
         }
     }
 
-    private boolean isClosed() {
+    public boolean isClosed() {
```

Comment:
review note: exposed to wrapper client to access `isClosed` rather than wrapper client duplicates and manages its own close feature.